app/experimenter/static/rapid/components/forms/ExperimentForm.tsx

```diff
@@ -200,31 +200,118 @@ export const BranchesForm: React.FC = () => {
   const formData = useExperimentState();
   const variants = formData.variants || [];
   return (
-    <div>
-      <table className="table table-bordered">
-        <thead>
-          <tr>
-            <th>Branch</th>
-            <th>Description</th>
-            <th>Content</th>
-            <th>Ratio</th>
-          </tr>
-        </thead>
-        <tbody>
-          {variants.map((variant, i) => (
-            <tr key={i}>
-              <td>
-                <code>{variant.name}</code>
-              </td>
-              <td>{variant.description}</td>
-              <td>
-                <code>(empty)</code>
-              </td>
-              <td>{variant.ratio}</td>
-            </tr>
-          ))}
-        </tbody>
-      </table>
+    <div style={{ width: "75%" }}>
+      <div className="mb-4">
+        <h5>Branches</h5>
+        <p>You can change the configuration of a feature in each branch.</p>
+      </div>
+      {[...variants]
+        // Sort control branch to the top
+        .sort((a, b) => (a.is_control < b.is_control ? 1 : -1))
+        .map((variant, i) => {
+          return (
+            <div key={i} className="card mb-4">
+              <ul className="list-group list-group-flush">
+                <li className="list-group-item">
+                  <div className="container">
+                    <div className="row">
+                      <div className="col">
```

Comment:
This is why I wanted to add `react-bootstrap` so we could just use the components instead of having to do it this way, then you get to use the props to pass config to them too.

Reply:
Yeah I can see how that would be nice. Especially for the closing tags (a million nested divs is annoying!)